src/aat/resources/features/F-048/S-133.td.json

```diff
@@ -0,0 +1,53 @@
+{
+  "_guid_": "S-133",
+  "_extends_": "S-000-Base-Success",
+  "title": "must return 400 if the sort direction in input parameters is not in ASC or DESC",
+  "specs": [
+    "a detailed profile in CCD",
+    "contains invalid Sort Direction",
+    "contains an error message : Unknown sort direction"
+  ],
+  "productName": "CCD Data Store",
+  "operationName": "Get the pagination metadata for a case data search for Case Worker",
+  "method": "GET",
+  "uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/pagination_metadata",
+  "request": {
+    "pathVariables": {
+      "uid": "[[DYNAMIC]]",
+      "jid": "AUTOTEST1",
+      "ctid": "AAT"
+    },
+    "queryParams": {
+      "sortDirection": "someInvalidSortDirection"
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 400,
+    "responseMessage": "Bad Request",
+    "headers": {
+      "X-Frame-Options": "[[ANY]]",
+      "Transfer-Encoding": "[[ANY]]",
+      "Cache-Control": "[[ANY]]",
+      "X-Content-Type-Options": "[[ANY]]",
+      "X-Xss-Protection": "[[ANY]]",
+      "Content-Encoding": "[[ANY]]",
+      "Vary": "[[ANY]]",
+      "Expires": "[[ANY]]",
+      "Pragma": "[[ANY]]",
+      "Date": "[[ANY]]",
+      "Content-Type": "[[ANY]]",
+      "Content-Length": "[[ANY]]"
```

Comment:
Why all are ANY?

Reply:
The headers were giving a hard time in pipeline. Hence changed to ANY.
Currently, I am using the headers defined in the base file rather than scenario data files.